satellite/satellitedb/repairqueue.go

```diff
@@ -32,10 +32,10 @@ func (r *repairQueue) Insert(ctx context.Context, seg *pb.InjuredSegment) (err e
 func (r *repairQueue) Select(ctx context.Context) (seg *pb.InjuredSegment, err error) {
 	defer mon.Task()(&ctx)(&err)
 	err = r.db.QueryRowContext(ctx, `
-		UPDATE injuredsegments SET attempted = timezone('utc', now()) WHERE path = (
+		UPDATE injuredsegments SET attempted = now() WHERE path = (
 			SELECT path FROM injuredsegments
-			WHERE attempted IS NULL OR attempted < timezone('utc', now()) - interval '1 hour'
-			ORDER BY attempted NULLS FIRST FOR UPDATE SKIP LOCKED LIMIT 1
```

Comment:
note:
- order by attempted with nulls first.

- for update withWith SKIP LOCKED, any selected rows that cannot be immediately locked are skipped. Skipping locked rows provides an inconsistent view of the data, so this is not suitable for general purpose work, but can be used to avoid lock contention with multiple consumers accessing a queue-like table